packages/date-picker/src/vaadin-month-calendar.js

```diff
@@ -414,16 +425,16 @@ class MonthCalendar extends FocusMixin(ThemableMixin(PolymerElement)) {
     }
   }
 
-  __isDayDisabled(date, minDate, maxDate) {
-    return !dateAllowed(date, minDate, maxDate);
+  __isDayDisabled(date, minDate, maxDate, isDateAvailable) {
+    return !dateAllowed(date, minDate, maxDate, isDateAvailable);
   }
 
-  __getDayAriaDisabled(date, min, max) {
+  __getDayAriaDisabled(date, min, max, isDateAvailable) {
```

Comment:
Doesn't look like the new function is properly passed to this method.

Also indicates a missing test, it looks like month calendar attributes like `part` and `aria-disabled` are currently covered by [snapshot tests](https://github.com/vaadin/web-components/blob/877d1d27545bb3f1affe47c304c34001da5f9e6d/packages/date-picker/test/dom/month-calendar.test.js#L29-L32), so a new snapshot test should be added there.